src/others/helpers/map-utils.ts

```diff
@@ -50,3 +27,34 @@ export const mapAidRequestsToFeatures = (mapData: RequestMapDataPoint[], locatio
   });
 };
 
+export type RegionMetadata = { [id: string]: {amount: number, description: string } };
```

Comment:
**issue/naming**
Why `Metadata`? It's just exactly the same data, but `GroupedByRegion`, isn't it? 
I find such inconsistencies extremely confusing.

Reply:
You mean the same data as MapDataPoint?

The two differences are:
 1) city_id -> region_id
 2) MapDataPoint is used for lists, while this one defines a dict.

So what is your exact proposal to name this type? 

If `GroupedByRegion` then i don't really like it (my not liking applies to existing names too).
The reason is GroupedBy* describes how the data was obtained and not what data does the object represent.
And I think the types should describe the data being stored, not how it was obtained.

For variable names it makes sense to have "groupedBy*" as part of the name, but not the type.

wdyt?